libs/chonky/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@rust/chonky",
+  "version": "0.0.0-private",
+  "private": true,
+  "license": "AGPL-3",
+  "scripts": {
+    "fix:clippy": "just clippy --fix",
+    "lint:clippy": "just clippy",
+    "test:unit": "cargo hack nextest run --feature-powerset --all-targets && cargo test --all-features --doc"
```

Comment:
`test:unit` will run in CI. We can add `test:integration` as well so that command would be executed as well.